cron-ghe-backup

```diff
@@ -1,4 +1,5 @@
 # ghe prod backups
+0 */2 * * 1-6 application find /data/ghe-production-data/ -maxdepth 2 -mindepth 2 -name incomplete -atime +1  -printf "%h\0" | xargs -0 rm -Rf
```

Comment:
`find` has a flag `-delete` that could have been used. I assume you want to print to log file as well, correct?

Reply:
No, because -delete flag cannot delete directories containing files